src/pages/tutorial/angular/step-1.mdx

```diff
@@ -0,0 +1,919 @@
+```
+
+### Add upstream remote
+
+Add a remote called `upstream` so we can eventually submit a pull request once you have completed this tutorial step.
+
+##### SSH:
+
+```bash
+git remote add upstream git@github.com:carbon-design-system/carbon-tutorial-angular.git
+```
+
+Or, if you prefer to use HTTPS instead of SSH with your remotes:
+
+##### HTTPS:
+
+```bash
+git remote add upstream https://github.com/carbon-design-system/carbon-tutorial-angular.git
+```
+
+Verify that your forked repository remotes are correct:
+
+```bash
+git remote -v
+```
+
+Your terminal should output something like this:
+
+```bash
+origin	[your forked repo] (fetch)
+origin	[your forked repo] (push)
+upstream	git@github.com:carbon-design-system/carbon-tutorial-angular.git (fetch)
+upstream	git@github.com:carbon-design-system/carbon-tutorial-angular.git (push)
+```
+
+### Branch
+
+Now that we have our repository set up, let's check out the branch for this tutorial step's starting point. Run the two commands:
+
+```bash
+git fetch upstream
+git checkout -b angular-step-1 upstream/angular-step-1
+```
+
+## Use starter app (recommended)
+
+<AnchorLinks>
+
+<AnchorLink>Add starter remote</AnchorLink>
+<AnchorLink>Pull starter code</AnchorLink>
+<AnchorLink>Remove starter remote</AnchorLink>
+<AnchorLink>Build and start</AnchorLink>
+<AnchorLink>Update UI Shell</AnchorLink>
+<AnchorLink>Create pages</AnchorLink>
+<AnchorLink>Add routing</AnchorLink>
+<AnchorLink>Submit pull request</AnchorLink>
+
+</AnchorLinks>
+
+### Add starter remote
+
+Add a temporary remote called `starter` so we can clone the [carbon-angular-starter](https://github.com/carbon-design-system/carbon-angular-starter).
+
+##### SSH:
+
+```bash
+git remote add starter git@github.com:carbon-design-system/carbon-angular-starter.git
+```
+
+Or, if you prefer to use HTTPS instead of SSH with your remotes:
+
+##### HTTPS:
+
+```bash
+git remote add starter https://github.com/carbon-design-system/carbon-angular-starter.git
+```
+
+### Pull starter code
+
+Now that we a have starter remote, we can rebase our branch to match whatever is in the starter app. Run the two commands:
+
+```bash
+git fetch starter
+git pull --allow-unrelated-histories starter master
+```
+
+### Remove starter remote
+
+Remove the temporary remote called `starter` since we have no use for it anymore.
+
+```bash
+git remote remove starter
+```
+
+## Build and start
+
+We have the repository forked to your GitHub account, cloned down to your machine, and the starting branch checked out. Next, install the Carbon app's dependencies with:
+
+```bash
+npm install
+```
+
+After the dependencies are installed, you can start the app with:
+
+```bash
+npm run start
+```
+
+Your app should now be running with the message: `** Angular Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **`
+
+That's it! Your browser should now resemble our [starter-app](https://carbon-angular-starter.netlify.com/).
```

Comment:
This URL only works for me if CMD-SHIFT-R reloading.

![image](https://user-images.githubusercontent.com/1691245/67425704-266fa680-f59e-11e9-9941-36cc8f83dc1f.png)

Reply:
I _think_ that may be due it a previous version installing itself as a PWA ... I don't have a good suggestion on how to fix that, it's something we've struggled with (the PWA code is disabled by default now)